src/components/scenes/SendScene2.tsx

```diff
@@ -1025,8 +1025,7 @@ const SendComponent = (props: Props) => {
       {({ insetStyle }) => (
         <>
           <StyledKeyboardAwareScrollView
-            notificationHeight={insetStyle.paddingBottom}
-            contentContainerStyle={{ ...insetStyle, paddingTop: 0, paddingBottom: theme.rem(5) + insetStyle.paddingBottom }}
```

Comment:
Hard-coded extra 5rem not needed: 
![image](https://github.com/EdgeApp/edge-react-gui/assets/90650827/5ae8ab33-9c59-4c62-9a3b-214cac7f7ad9)